src/desktop/assets/analytics.coffee

```diff
@@ -14,11 +14,19 @@ Events.onEvent (data) =>
   analytics.track data.action, _.omit data, 'action'
 
   # Send Reaction's read more as a Parsely page view
-  if data.action is 'Clicked read more' and window.PARSELY
-    window.PARSELY.beacon.trackPageView
-      url: location.href,
-      js: 1,
-      action_name: 'infinite'
+  if data.action is 'Clicked read more'
+    if window.PARSELY
+      window.PARSELY.beacon.trackPageView
+        url: location.href,
+        js: 1,
+        action_name: 'infinite'
+    if window.Sailthru
+      window.Sailthru.track
+        domain: 'horizon.artsy.net',
+        spider: true,
+        track_url: true,
+        url: sd.APP_URL + '/' + location.pathname,
+        use_stored_tags: true
```

Comment:
This was never added into new article tracking. Pulled this from https://github.com/artsy/force/pull/2210/files#diff-f34aebe5846b1370b3bd55d88c7aea14L86